### PR TITLE
Login & signUp Pages

### PR DESCRIPTION
**Pull request by aboodswiti**
- create multiStip Login page and split the password form

- create checkAccount controller in the back-end to check login email if exist in data-base

- create multiStip signUp page 

- create helper folder for signUp front-end validation form using Yup

- create formsStyles file for login and signup styles

### CODE REVIEW

File: client/src/Components/Layouts/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  Container,
+  Box,
+} from "@material-ui/core";
+import Chip from "@material-ui/core/Chip";
+import MailOutlineIcon from "@material-ui/icons/MailOutline";
+import useStyles from "../../../Theme/FormsStyles";
+import axios from "axios";
+import PasswordForm from './PasswordForm'
+export default function Login() {
+  const classes = useStyles();
```
Comment (someyaaltous) [inline client/src/Components/Layouts/Login/Login.js:15]: change all function to arrow function, see this #67

**Comment by someyaaltous** [comment]
Solve conflict, change all function to arrow function, change all Arabic message to English

**Comment by someyaaltous** [comment]
Added header to your page

**Comment by someyaaltous** [comment]
Use Eslint and prettier in your code.

File: client/src/Components/Layouts/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  };
+
+  return (
+    <Grid container direction="column"  className={classes.root}>
+        <SecondHeader pageName="Access" HideIcon='true' />
+
+    <Container className={classes.Container}>
+
+
+      {accountStatus !== "Exist" ? (
```
[the remark this answers is not in the document]
Comment (aboodswiti) [reply client/src/Components/Layouts/Login/Login.js:86]: what you mean of "magic strings" ?

File: server/controllers/logIn.js [hunk 3]
```diff
@@ -2,14 +2,21 @@ const getPassword = require('../database/queries/getPassword')
 const createToken = require('../middlewares/createToken')
 
 const login = (req, res) => {
-  const { email, password } = req.body
+  const { email, password } = req.body.values
+console.log('req.body',req.body.values);
 
-  if (email.trim().length === 0 || password.trim().length === 0) {
```
Comment (someyaaltous) [inline server/controllers/logIn.js:7]: Delete comment here

File: client/src/Components/Layouts/signUp/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+          handleMouseDownPassword={handleMouseDownPassword}
+          handleMouseDownPassword={handleMouseDownPassword}
+          message={message}
+          next={next}
+          showPassword={showPassword}
+          checkAccount={checkAccount}
+        />
+        {next ? (
+          <Link to='/login' className='text-link'>
+            <p className={classes.text}>
```
Comment (someyaaltous) [inline client/src/Components/Layouts/signUp/SignUp.js:85]: Delete this here  
```
              {' '}

```

**Pull request merged**